src/analyzer/protocol/ftp/FTP.h

```diff
@@ -22,8 +22,9 @@ class FTP_Analyzer final : public analyzer::tcp::TCP_ApplicationAnalyzer {
 protected:
     analyzer::login::NVT_Analyzer* nvt_orig;
     analyzer::login::NVT_Analyzer* nvt_resp;
-    uint32_t pending_reply;     // code associated with multi-line reply, or 0
-    std::string auth_requested; // AUTH method requested
+    uint32_t pending_reply;       // code associated with multi-line reply, or 0
+    std::string auth_requested;   // AUTH method requested
+    bool switched_to_tls = false; // starttls active
```

Comment:
This is called `tls_active` in quite a few analyzers  and also `starttls` in case you'd be up for aligning.